core/new-gui/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
+root = true
```

Comment:
The website says that VS code has to install an extension to support editor config. If so, can you change your ppt on frontend to include that this extension should also be installed.

Reply:
Sure, this is a nice suggestion.